mozilla-release/browser/components/BrowserGlue.jsm

```diff
@@ -3061,6 +3061,17 @@ BrowserGlue.prototype = {
     // set true if we need to show whats new page
     Services.prefs.setBoolPref("browser.migration.showWhatsNew", false);
 
+    // CLIQZ-SPECIAL: fallback to  default content process count in case
```

Comment:
@tsl143 let's put it into a separate function cliqz_clearProcessCount.
What do you think?